### PR TITLE
Ts/3433 speed quality button

### PR DESCRIPTION
**Pull request by dvince2**
# Description

* Simulation presets.

Resolves #(issue)

### CODE REVIEW

File: packages/client/hmi-client/src/components/workflow/ops/simulate-ciemss/tera-simulate-ciemss.vue [hunk 1]
```diff
@@ -266,6 +305,18 @@ const chartProxy = chartActionsProxy(props.node, (state: SimulateCiemssOperation
 	emit('update-state', state);
 });
 
+const setPresetValues = (data: PresetTypes) => {
+	console.log(data);
```
Comment (Tom-Szendrey) [inline packages/client/hmi-client/src/components/workflow/ops/simulate-ciemss/tera-simulate-ciemss.vue:309]: no need for this console log

**Review by Tom-Szendrey — APPROVED** [review]
IMO the labels should maybe be debug and preset 
-Debug as it should only be used for this 
-Preset as a generic wording
What "Normal" means can vary :P 
But this is more of a design thing than anything and a super minor change in future if we want

**Comment by mwdchang** [comment]
> IMO the labels should maybe be debug and preset -Debug as it should only be used for this -Preset as a generic wording What "Normal" means can vary :P But this is more of a design thing than anything and a super minor change in future if we want

Yep this is the issue here, we have fast/debug and not fast/debug ... so there is really only one preset but we are modelling it as two.

**Review by YohannParis — APPROVED** [review]

File: packages/client/hmi-client/src/components/workflow/ops/simulate-ciemss/tera-simulate-ciemss.vue [hunk 2]
```diff
@@ -255,6 +282,18 @@ const outputs = computed(() => {
...
 	return [];
 });
+
+const presetType = computed(() => {
+	if (numSamples.value === speedValues.numSamples && method.value === speedValues.method) {
+		return PresetTypes.Fast;
+	}
+	if (numSamples.value === qualityValues.numSamples && method.value === qualityValues.method) {
+		return PresetTypes.Normal;
+	}
```
Comment (mwdchang) [inline packages/client/hmi-client/src/components/workflow/ops/simulate-ciemss/tera-simulate-ciemss.vue:292]: There is only "Fast" mode, everything that is not Fast is Normal

**Review by mwdchang — APPROVED** [review]

**Pull request merged**